src/components/kbd.html

```diff
@@ -1,18 +0,0 @@
-<div class="p-20 text-slate max-w-lg">
```

Comment:
I think this is the component we discussed with lauren that's used on Data Stories, guessing we don't want to delete it?